src/longopt/grdinfo_inc.h

```diff
@@ -25,41 +25,35 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'C', "oneliner",
+	          "n,t",               "numeric,name_at_end",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'D', "tiles",
 	          "",                  "",
+	          "i",                 "ignore_empty",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'E', "extreme|extrema",
+	          "x,y",               "x,y",
+	          "l,L,u,U",           "min,minpos,max,maxneg",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'F', "geographic",        "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "force_remote",      "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'I', "minmax_region",
+	          "b,i,o,r",           "polygon,imgexact,oblique,wesn",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
+	{ 0, 'L', "force_scan",
+	          "0,1,2,p,a",         "scandata,medianL1,meanplus,modeLMS,all",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
+	{ 0, 'M', "minmax_pos",
+	          "c,f",               "conditional,force",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'T', "",
-	          "",                  "",
+	{ 0, 'T', "minmax_gridval",
```

Comment:
Being a `grd`, I think `minmax` has to refer to the grid values. I find `minmax_gridval` too long and breaks with GMT.jl

Reply:
I agree, let us avoid mentioning grid since the values obviously come from the grid. So --minmax.